testnet/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -585,10 +585,41 @@ impl UTXO {
     }
 
     pub fn get_sat_amount(&self) -> u64 {
-        let res = self.amount * 10e7;
-        res as u64
+        UTXO::serialized_btc_to_sat(&self.amount)
     }
 
+    pub fn serialized_btc_to_sat(amount: &str) -> u64 {
+        let comps: Vec<&str> = amount.split(".").collect();
+        match comps[..] {
+            [lhs, rhs] => {
+                let base: u64 = 10;
+                let sat_decimals = 7;
+                let btc_to_sat = base.pow(7);
```

Comment:
This should be `base.pow(8)`

Reply:
Yeah, can you refresh? :)

Reply:
Ah lol forgot I was looking at just the commit ;)